tests/test_file_set_processor_with_ocr.py

```diff
@@ -0,0 +1,167 @@
+from pathlib import Path
+
+import pytest
+
+from dor.providers.file_system_file_provider import FilesystemFileProvider
+from dor.providers.process_basic_image import (
+    Command,
+    CompressSourceImage,
+    CreateTextAnnotationData,
+    ExtractImageText,
+    ExtractImageTextCoordinates,
+    FileSetIdentifier,
+    Input,
+    process_basic_image,
+)
+
+@pytest.fixture
+def file_set_identifier() -> FileSetIdentifier:
+    return FileSetIdentifier(project_id="collid", file_name="quick-brown.tiff")
+
+
+@pytest.fixture
+def input_path() -> Path:
+    return Path("tests/fixtures/test_basic_copy_with_ocr")
+
+
+@pytest.fixture
+def output_path() -> Path:
+    file_provider = FilesystemFileProvider()
+    output_path = Path("tests/test_process_image_with_ocr")
```

Comment:
Thus far we haven't been very consistent with how we name our fixture input and output directories. How should we name them going forward? At the very least it would be nice if the file set processor stuff had the same prefixes.